api/models/Tag.js

```diff
@@ -53,6 +53,12 @@ const addToCommunity = (community, tag, user_id, trx) => {
       user_id: user_id,
       created_at: new Date()
     }).save({}, {transacting: trx})
+    .then(() => new TagFollow({
```

Comment:
I'll have to add tag_follows for existing community_tags in the db before merging this